app/init.php

```diff
@@ -208,6 +208,9 @@
 const MESSAGE_TYPE_EMAIL = 'email';
 const MESSAGE_TYPE_SMS = 'sms';
 const MESSAGE_TYPE_PUSH = 'push';
+// API key types
+const API_KEY_STANDARD = 'standard';
+const API_KEY_DYNAMIC = 'dynamic';
```

Comment:
What is the size of the API key ? Will we hit any limits on the API key length during any stage? Here are the scenarios I can think of.
1. When storing the encrypted API key in database
2. When sending the API key through the header

Reply:
We can also use smaller prefixes like stripe. I believe they use `pk` for production key and `dk` for development key

Reply:
i dont like `pk` and `dk`, and openAI's `sk`.. I never know what those mean. to me `sk` means `Slovakia` 😅
I would say going verbose isn't problem, considering there is `_` so it is consider one word when you double-click to copy

Reply:
JWT keys arent stored in DB, should be fine.
Header can go up to KBs of size, also shouldn't be an issue. If we ever get some kind of report, I am sure we can shorten JWT key with some configuration.

Normla keys will only get few letters prefix, shouldn't be length issue.